### ISSUE TITLE
Bug: Cache cleaning through accessibility does not work

### ISSUE THREAD

**Issue by lordballiwns** [d4rken/sdmaid-public#3861]
### What happened?
The cache cleaning service starts but does not clear the cache automatically. 

### What did you expect to happen?
That the service will clear the cache used on the system. To avoid doing it manually. 

### How can I reproduce the issue?
Using any device with Android 7.0 or higher. But lower than Android 8.0. 

### Comments
I love the app to the degree that I buy the app. However, for some time now, it no longer works on my Android device (or another with Android 7) since the beginning of the year.
### SD Maid
Version: `4.15.14 (41514)`
Pro: `isPro=true || isComplete=true (unl=true, iap=false, acc=false)`

### Device
Root: `RootContext(rootState=Root(state=UNAVAILABLE)`
Fingerprint: `LANIX/Ilium_X520/Ilium_X520:7.0/NRD90M/X520_TELCEL_SW_19:user/release-keys`

**Comment by lordballiwns** [comment]
Attached debug report link. Permanent link. https://mega.nz/folder/WMwnETJA#6PZ79FjWjgFQ2LeQZk6W_g

**Comment by lordballiwns** [comment]
![Screenshot_2020-08-25_07-28-56](https://user-images.githubusercontent.com/44513582/91174323-be4d8d80-e6a4-11ea-9c28-1f686842ca82.png)

**Comment by ceto61** [comment]
![Screenshot_20200825-220157](https://user-images.githubusercontent.com/70232305/91216524-cb0ec780-e71e-11ea-80ff-d71fc7343569.png)
Uygulama temizliği yapmıyor

**Comment by d4rken** [comment]
Can you show me a screenshot of your appcleaner options?

**Comment by lordballiwns** [comment]
Yes. In one more moment I upload catches.

**Comment by lordballiwns** [comment]
![Screenshot_2020-08-29_11-47-35](https://user-images.githubusercontent.com/44513582/91642052-dcd9be80-e9ed-11ea-9c79-2c74cd7badaa.png)
![Screenshot_2020-08-29_11-47-49](https://user-images.githubusercontent.com/44513582/91642054-df3c1880-e9ed-11ea-9aad-d73c6a3ec390.png)
![Screenshot_2020-08-29_11-46-36](https://user-images.githubusercontent.com/44513582/91642056-e105dc00-e9ed-11ea-8cb6-84e935ed8d28.png)
![Screenshot_2020-08-29_11-48-19](https://user-images.githubusercontent.com/44513582/91642057-e2370900-e9ed-11ea-92f7-df0c669aebf2.png)

this has been the configuration in SD MAID

**Comment by lordballiwns** [comment]
This is accessibility. How you can see is active. 
![Screenshot_2020-08-29_11-46-12](https://user-images.githubusercontent.com/44513582/91642104-35a95700-e9ee-11ea-996e-5690988f9251.png)

**Comment by lordballiwns** [comment]
And finally. These are the manual cache clearing option. How can you observe. It is different from Android 8, 9 and 10. Hence the automatic cache clearing error occurs. 
![Screenshot_2020-08-29_11-49-25](https://user-images.githubusercontent.com/44513582/91642159-a6507380-e9ee-11ea-9789-5672e8462bc2.png)
![Screenshot_2020-08-29_11-49-07](https://user-images.githubusercontent.com/44513582/91642160-a781a080-e9ee-11ea-82df-5cfe90b517fd.png)

**Comment by lordballiwns** [comment]
If you wish. I can show you a live screen of my smartphone using Team Viewer. So you can observe the error.

**Comment by d4rken** [comment]
Thanks, I think it's okay with the bug report for now.
I'll need some time to investigate, and will comment here if I need more info. Thanks!

**Comment by d4rken** [comment]
Should be fixed with v5.0.7.

Happy holidays and stay healthy!
